styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -681,11 +683,17 @@ private static DockerRunner createDockerRunner(
       LOG.info("Creating LocalDockerRunner");
       return closer.register(DockerRunner.local(scheduler, stateManager));
     } else {
+      final String styxEnvironment;
+      if (!config.hasPath(STYX_ENVIRONMENT)) {
+        styxEnvironment = STYX_ENVIRONMENT_PRODUCTION;
```

Comment:
is production a good/safe default?

Reply:
No idea. ;) Maybe `development` or `NA`?